docs/data-collect-prep/ingesting_data.md

```diff
@@ -0,0 +1,101 @@
+(ingesting_data)=
+# Ingesting data
+
+MLRun provides a set of tools and capabilities to streamline the task of data ingestion and processing. For an 
+end-to-end framework for data processing, management and serving MLRun has the feature-store capabilities, which are
```

Comment:
For an end-to-end framework for data processing, management, and serving, MLRun has the feature-store capabilities...